packages/backend/src/open_payments/quote/service.ts

```diff
@@ -121,13 +121,19 @@ async function createQuote(
       receiver
     })
 
+    const quoteAssetId = paymentPointer.assetId
```

Comment:
I guess it's not necessary, especially since they arent multiple assetId's in play here. I assigned it to a variable because the `setFee` `assetId` and `assetId` inserted into the quote should be the same.